crypto/threads_pthread.c

```diff
@@ -162,6 +166,11 @@ int CRYPTO_atomic_add(int *val, int amount, int *ret, CRYPTO_RWLOCK *lock)
         *ret = __atomic_add_fetch(val, amount, __ATOMIC_ACQ_REL);
         return 1;
     }
+# elif defined(__sun) && (defined(__SunOS_5_10) || defined(__SunOS_5_11))
```

Comment:
I assume this also works with later versions?

Reply:
There will never be Solaris 12 so yes. All Solaris 11.x versions will have the `__SunOS_5_11` defined.

Reply:
That question will most probably be asked again in the future, so a comment saying this would be a good thing.

Reply:
okay, added.